protocol/avdl/keybase1/simple_fs.avdl

```diff
@@ -268,4 +268,13 @@ protocol SimpleFS {
    Get sync status.
    */
   FSSyncStatus simpleFSSyncStatus();
+
+  @lint("ignore")
+  record SimpleFSGetHTTPAddressAndTokenResponse {
+    string address;
+    string token;
+  }
+
+  @lint("ignore")
+  SimpleFSGetHTTPAddressAndTokenResponse SimpleFSGetHTTPAddressAndToken();
```

Comment:
Can you add comments here about how this function should be used?  One token per session, one per file access, etc?  Also, presumably the caller will have to call this again if they ever have to re-connect to simplefs (say if KBFS restarts on desktop); that should be documented somewhere too.